modules/openideal_faq/modules/openideal_faq_content/openideal_faq_content.module

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file Contains main hooks for ofc module.
+ */
+
+use Drupal\node\NodeInterface;
+
+/**
+ * Implements hook_ENTITY_TYPE_insert().
+ *
+ * Dirty save path alias that is hardcoded to the node uuid that is
+ * imported with this module, because of update_helper cannot save
+ * path alias.
+ */
+function openideal_faq_content_node_insert(NodeInterface $node) {
+  if ($node->uuid() == '62e92783-c1a6-4cc2-bd50-198884f083ec') {
+    \Drupal::service('path.alias_storage')->save("/node/" . $node->id(), "/faq", "en");
```

Comment:
Let's use single quotes to be consistent.

Reply:
Done.